lib/features/push_notification/data/local/fcm_cache_manager.dart

```diff
@@ -56,4 +58,9 @@ class FCMCacheManager {
       throw NotFoundDeviceIdException();
     }
   }
+
+  void deleteFcmToken() async {
+    FcmController.instance.deleteFcmToken();
```

Comment:
is it ok to put it here? IMO, `FCMCacheManager` only work with cache (local) stuff.